samples/apps/copilot-chat-app/webapi/Options/PromptsOptions.cs

```diff
@@ -61,89 +54,83 @@ public PromptSettings(PromptsConfig promptsConfig)
     /// </summary>
     internal double DocumentMemoryMinRelevance { get; } = 0.8;
 
-    internal string KnowledgeCutoffDate => this._promptsConfig.KnowledgeCutoffDate;
-    internal string InitialBotMessage => this._promptsConfig.InitialBotMessage;
-
-    // System commands
-    internal string SystemDescriptionPrompt => this._promptsConfig.SystemDescription;
-    internal string SystemResponsePrompt => this._promptsConfig.SystemResponse;
-
-    // Intent extraction commands
-    internal string SystemIntentPrompt => this._promptsConfig.SystemIntent;
-    internal string SystemIntentContinuationPrompt => this._promptsConfig.SystemIntentContinuation;
-
+    // System
+    [Required, NotEmptyOrWhitespace] public string KnowledgeCutoffDate { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string InitialBotMessage { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string SystemDescription { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string SystemResponse { get; set; } = string.Empty;
     internal string[] SystemIntentPromptComponents => new string[]
     {
-        this.SystemDescriptionPrompt,
-        this.SystemIntentPrompt,
+        this.SystemDescription,
+        this.SystemIntent,
```

Comment:
I feel like the name `SystemIntent` and some others may cause confusion if we just want to remove the word `prompt` in the variable names.
Some suggestions:
`SystemDescription` -> `ChatDescription`
`SystemResponse ` -> `ChatResponseInstruction`
`SystemChatContinuation` -> `ChatContinuation`
`SystemIntent` -> `IntentDescription`
`SystemIntentContinuation` -> `IntentContinuation`
...

Open to other names

Reply:
Let's make a pass at these changes in a separate PR.